IM_calculation/Advanced_IM/advanced_im_config.yaml

```diff
@@ -6,5 +6,9 @@
 
 Steel_MF_5Story:
   script_location: 'Models/Steel_MF_5Story/run.py'
+SAC_Steel_MF_3Story:
+  script_location: 'Models/SAC_Steel_MF_3Story/run.py'
+SAC_Steel_MF_9Story:
+  script_location: 'Models/SAC_Steel_MF_9Story/run.py'
 SAC_Steel_MFs:
```

Comment:
Remove these two lines. This is the two new models now implemented.

Reply:
but the new models are added in to the repo?
and without these two lines it will not run the new models.